src/utils/constants.ts

```diff
@@ -9,3 +10,10 @@ export const ONE_ETHER = BigInt.fromString("1000000000000000000");
 export const NONE = "NONE";
 export const COMETHSPACESHIP = "COMETHSPACESHIP";
 export const MAX_UINT256 = BigInt.fromString("115792089237316195423570985008687907853269984665640564039457584007913129639935");
+
+export const ALCHEMICA_TYPE_TO_ADDRESS = [FUD_TOKEN_ADDRESS, FOMO_TOKEN_ADDRESS, ALPHA_TOKEN_ADDRESS, KEK_TOKEN_ADDRESS];
+
+export const REWARDS_DISTRIBUTED_EVENT = "RewardsDistributed";
+export const ALCHEMICA_CLAIMED_EVENT = "AlchemicaClaimed";
+export const CHANNEL_ALCHEMICA_EVENT = "ChannelAlchemicaClaimed";
+export const TRANSFER_TOKENS_TO_GOTCHI_EVENT = "TransferTokensToAavegotchi";
```

Comment:
Using more user friendly names. This way, the front-end/back-end won't have to parse the subgraph response, they will just show the `eventName`
```suggestion
export const REWARDS_DISTRIBUTED_EVENT = "Player Rewards";
export const ALCHEMICA_CLAIMED_EVENT = "Farming";
export const CHANNEL_ALCHEMICA_EVENT = "Channeling";
export const TRANSFER_TOKENS_TO_GOTCHI_EVENT = "Vortex";
```